core/build.gradle

```diff
@@ -54,6 +86,9 @@ dependencies {
     compile ([group: 'org.apache.httpcomponents', name: 'httpclient', version: '4.5.3'])
     compile ([group: 'com.google.guava', name: 'guava', version: '20.0'])
     compile ([group: 'com.google.code.gson', name: 'gson', version: '2.8.2'])
+    compile ([group: 'com.google.protobuf', name:'protobuf-java', version:'3.6.1'])
```

Comment:
@littleaj I hope we have the required permission for all these three libraries from OSS before shipping. Further just a heads-up I might have to soon change the MetricTelemetry and it's associated data types (the Schema has been changed a bit there, and would need to run the bond again). At that time we might need Proto files to be regenerated. Do they generate on every build now? Also, what are the implications if we need to regenerate proto files. Do we need to release LF and SDK again at same time ?

Reply:
I'm working on approval for the library. The gradle tool is already approved.